jekyll/_cci2/rerun-failed-tests.adoc

```diff
@@ -309,11 +291,11 @@ Ensure that you are passing a list of test filenames (or classnames) *via stdin*
 
 ---
 
-`circleci tests run` expects input to be space or newline delimited.  If your test file names have spaces in them, this may pose a problem, especially if you are using `pytest` which may generate names with whitespace.  One possible workaround is to use specific IDs for the tests with whitespace in their names using the instructions link:https://docs.pytest.org/en/7.1.x/example/parametrize.html#set-marks-or-test-id-for-individual-parametrized-test[here].  
+`circleci tests run` expects input to be space or newline delimited.  If your test file names have spaces in them, this may pose a problem, especially if you are using `pytest` which may generate names with whitespace.  One possible workaround is to use specific IDs for the tests with whitespace in their names using the instructions link:https://docs.pytest.org/en/7.1.x/example/parametrize.html#set-marks-or-test-id-for-individual-parametrized-test[here].
 
-If your job runs tests in parallel and persists files to a workspace, you may see a parallel run on a re-run that fails because the `persist_to_workspace` step could not find any contents in the directory that was specified. This may happen because the parallel run will not always execute tests on a re-run if there are not enough tests to be distributed across all parallel runs.
+If your job runs tests in parallel and persists files to a workspace, you may see a parallel run on a rerun that fails because the `persist_to_workspace` step could not find any contents in the directory that was specified. This may happen because the parallel run will not always execute tests on a rerun if there are not enough tests to be distributed across all parallel runs.
```

Comment:
I wonder if it makes sense to link to this section from the Known Limitations point above re: parallelism? e.g. "see the troubleshooting section for avoiding parallel run failure" or something to that effect

Reply:
@kbatuigas I'm not sure what you're suggesting here, probably my Friday brain. Let's discuss at standup.

Reply:
I think I've done this now